### PR TITLE
Adding grafana dashboard

### PR DESCRIPTION
**Pull request by bardielle**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,3 +106,61 @@ metadata:
...
+    auth.anonymous: {}
+    security:
+      admin_password: secret
+      admin_user: root
+  ingress:
+    enabled: true
+ ``` 
+-  Connecting Prometheus to our Custom Grafana
+   -  Grant cluster-monitoring-view cluster role to the  grafana-serviceaccount service account, that was created alongside the Grafana instance\
+   `oc adm policy add-cluster-role-to-user cluster-monitoring-view -z grafana-serviceaccount`
```
[the remark this answers is not in the document]
Comment (bardielle) [reply README.md:131]: As I understand, role binding is for user or group 
https://docs.openshift.com/container-platform/3.7/admin_guide/manage_rbac.html#managing-role-bindings

**Review by ydayagi — APPROVED** [review]

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (gciavarrini) [reply README.md:131]: I can't execute this step correctly.

I installed Grafana from the OperatorHub on `k4e-operator-system` namespace, and then I created a Grafana instance as described.

Then:
```bash
➜  ~  oc login -u kubeadmin  https://api.crc.testing:6443
Logged into "https://api.crc.testing:6443" as "kubeadmin" using existing credentials.

You have access to 65 projects, the list has been suppressed. You can list all projects with 'oc projects'

Using project "default".
➜  ~  oc adm policy add-cluster-role-to-user cluster-monitoring-view -z grafana-serviceaccount
Warning: role 'cluster-monitoring-view' not found
Warning: ServiceAccount 'grafana-serviceaccount' not found
clusterrole.rbac.authorization.k8s.io/cluster-monitoring-view added: "grafana-serviceaccount"
```
The clusterrole `cluster-monitoring-view` hasn't been created
```bash
➜  k4e-operator (bardielle/adding_grafana) oc get clusterrole | grep cluster-monitoring-view                                                     
openshift-cluster-monitoring-view                                           2021-11-07T04:45:20Z
```
The grafana-serviceaccount is available only in the `k4e-operator-system` namespace
```bash
➜  k4e-operator (bardielle/adding_grafana) oc get serviceaccount                                                                                 ✭
NAME       SECRETS   AGE
builder    2         44d
default    2         44d
deployer   2         44d
➜  k4e-operator (bardielle/adding_grafana) oc get serviceaccount -n k4e-operator-system                                                          ✭
NAME                                  SECRETS   AGE
builder                               2         34m
default                               2         34m
deployer                              2         34m
grafana-operator-controller-manager   2         33m
grafana-serviceaccount                2         31m
k4e-operator-controller-manager       2         34m
prometheus-k8s                        2         3m12s
prometheus-operator                   2         3m12s
```

I'm using CodeReady Container:
```
➜  k4e-operator (bardielle/adding_grafana) crc status                                                                                            ✭
CRC VM:          Running
OpenShift:       Running (v4.9.5)
Disk Usage:      14.53GB of 32.74GB (Inside the CRC VM)
Cache Usage:     16.75GB
```

@ydayagi can you execute `oc adm policy add-cluster-role-to-user cluster-monitoring-view -z grafana-serviceaccount` without warnings?

[the remark this answers is not in the document]
Comment (ydayagi) [reply README.md:131]: i'll try

[the remark this answers is not in the document]
Comment (bardielle) [reply README.md:131]: Maybe the issue is that you're using crc? 
If you're adding manually the  cluster-monitoring-view cluster role: 
```
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRole
metadata:
  name: cluster-monitoring-view
rules:
- apiGroups:
  - ""
  resources:
  - namespaces
  verbs:
  - get

```

[the remark this answers is not in the document]
Comment (bardielle) [reply README.md:131]: Maybe the issue is that you're using crc? 
If you add manually the  cluster-monitoring-view cluster role does it work for you?
```
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRole
metadata:
  name: cluster-monitoring-view
rules:
- apiGroups:
  - ""
  resources:
  - namespaces
  verbs:
  - get

```

[the remark this answers is not in the document]
Comment (gciavarrini) [reply README.md:131]: Yes, it seems that the issue was due CodeReady Container. It disables Cluster Monitoring Operator by default.
Enabling it resolves the issue.

**Review by gciavarrini — APPROVED** [review]

**Review by jakub-dzon — APPROVED** [review]

**Pull request merged**